app/Auth/AlbumAuthorisationProvider.php

```diff
@@ -323,7 +323,7 @@ public function applyBrowsabilityFilter(AlbumBuilder $query, ?Album $origin = nu
 		}
 
 		// ... such that there are no blocked albums on the path to the album.
-		if (AccessControl::is_admin()) {
+		if (Auth::getUser()?->isAdmin() === true) {
```

Comment:
```suggestion
		if (Auth::getUser()?->isAdmin()) {
```

`isAdmin` returns a boolean.

Reply:
No. `getUser()` returns `null` or `User`.
with the null-safe accessor, if Auth::getUser() returns null we get null which would imply a type conversion (which we don't want as per our phpStan strict rules ;) ).

Surprisingly, phpstan is not picking that one up either...

Reply:
Good point and yes that is surprising.